htdocs/lockaccount.php

```diff
@@ -6,6 +6,7 @@
 $result = "";
 $dn = "";
 $password = "";
+$comment = $_POST["comment"];
```

Comment:
Like other POST params, you should check the presence of the param with isset (see example with "dn" just below)